### DISCUSSION TITLE
wich zip file to download for latest stable lhc release?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1842] [General]
to download zip file of lhc i have 2 links:
https://github.com/remdex/livehelperchat/archive/master.zip
https://github.com/LiveHelperChat/livehelperchat/archive/refs/heads/master.zip

which one should you take for the latest stable release or are they always the identical files?

**Comment by remdex** [comment]
I suppose, they are, just just for version number :)

I would go with
https://github.com/LiveHelperChat/livehelperchat/archive/refs/heads/master.zip
